tests/api-tests/admin-meta.test.ts

```diff
@@ -133,6 +133,7 @@ test(
       },
     }),
   })(async ({ context }) => {
+    const gql = String.raw;
```

Comment:
```suggestion
    const gql = ([content]: TemplateStringsArray) => content;
```
sorry, an annoying nit, `String.raw` happens to not cause problems here but it's basically never what you actually want (because the `raw` means escape sequences won't be interpreted and will instead be in the returned string)